docs/overview.md

```diff
@@ -0,0 +1,22 @@
+# ScalarDB Overview
+
+## What is ScalarDB?
+
+ScalarDB is an HTAP engine for diverse databases. It achieves ACID transactions and real-time analytics across databases to simplify the complexity of managing multiple databases. Existing solutions are designed to run only transactions or analytical queries across heterogeneous databases or both on homogeneous databases.
```

Comment:
If we say an HTAP engine here, it might be good to add an analytical flavor in the use cases section since currently nothing related to analytics is mentioned there.

> to run only transactions or analytical queries across heterogeneous databases or both on homogeneous databases.

Also, this part is a bit hard to follow for me. I don't have a good idea, but something like the following? It's a little redundant, though...

Existing solutions are either of the following, although ScalarDB can run both transactional and analytical queries on heterogeneous databases.
- Run transactions only across heterogeneous databases
- Run analytical queries only across heterogeneous databases
- Run both transactional and analytical queries on homogeneous databases

Reply:
Thank you!
Good point. 
Let me update it based on your suggestions.